docs/docsite/rst/user_guide/playbooks_templating.rst

```diff
@@ -1,14 +1,15 @@
 .. _playbooks_templating:
 
+*******************
 Templating (Jinja2)
-===================
+*******************
 
-As already referenced in the variables section, Ansible uses Jinja2 templating to enable dynamic expressions and access to variables.
-Ansible greatly expands the number of filters and tests available, as well as adding a new plugin type: lookups.
+Ansible uses Jinja2 templating to enable dynamic expressions and access to variables. Ansible includes a lot of specialized filters and tests for templating. You can use all the standard filters and tests included in Jinja2 as well. Ansible also offers a new plugin type: lookups.
```

Comment:
Should "lookups" be highlighted, or possibly linked to lookup docs?

Reply:
ah, yep, good idea

Reply:
done